templates/.rubocop.yml

```diff
@@ -36,11 +36,28 @@ Style/HashTransformValues:
 Style/HashAsLastArrayItem:
   Enabled: false
 
+Metrics/MethodLength:
+  CountAsOne:
+    - array
+    - hash
+    - heredoc
+  Exclude:
+    - 'db/**/*'
```

Comment:
as suggested in https://github.com/renuo/applications-setup-guide/pull/268
I would also like to add CountAsOne to MethodLength and not only to ClassLength